Automation/task2/team2/deployment.ps1

```diff
@@ -0,0 +1,73 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory=$true)]
+    [string]
+    $SubscriptionId,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $rgName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $location,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $TenantId,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $ServicePrincipalId,
+
+    [Parameter(Mandatory=$true)]
+    [securestring]
+    $ServicePrincipalPassword,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $TemplatefilePath,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $FunctionName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $StorageAccountName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $AppServiceplaneName,
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $DeployName
+)
+#open log
+$prefix = "jin-templa-deployment"
```

Comment:
This is pretty reasonable to add a logging function which can help with debugging. Thumbs up for this.  I am guessing we are overwriting the same log file again and again on each new deployment? or is the new input log appended to the previous log. But this is just a follow up question. All in all this looks good and I approve the merge.

Reply:
I deployed it successfully and  created the required output , I didn't see any error, so I approve the merge